Casks/t/tsh.rb

```diff
@@ -8,7 +8,7 @@
   desc "SSH server for teams managing distributed infrastructure"
   homepage "https://goteleport.com/"
 
-  deprecate! date: "2024-11-18", because: :unmaintained, replacement: "teleport"
+  deprecate! date: "2024-11-18", because: :unmaintained, replacement_formula: "teleport"
```

Comment:
```suggestion
  deprecate! date: "2024-11-18", because: :unmaintained, replacement_cask: "teleport"
```

`teleport` also exists as a cask and probably makes more sense, since we are replacing a cask.

Reply:
Thank you for the feedback